courses-portlet-webapp/src/main/webapp/css/degree-progress.css

```diff
@@ -1,215 +0,0 @@
-.degree-progress .fl-flex-2 {
```

Comment:
No problem -- it's completely cool and normal to handle 1 portlet at a time... but we need to keep degree-progress working in the mean time.